src/renderers/WebGLRenderer.js

```diff
@@ -1582,6 +1582,7 @@ function WebGLRenderer( parameters ) {
 			// wire up the material to this renderer's lighting state
 
 			uniforms.ambientLightColor.value = lights.state.ambient;
+			uniforms.sphericalHarmonicsValues.value = lights.state.sphericalHarmonicsValues;
```

Comment:
I think you can drop the `Values`-suffix to keep the code consistent with the others.

`lights.state.directionalValues` is called `lights.state.directional`, same as the rest.